### PR TITLE
DDF for Tuya door sensor ZG-102Z (_TZE200_pay2byax)

### PR DESCRIPTION
**Pull request by Smanar**
See https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6597

- Product name: Zigbee Door Sensor ZG-102Z
- Manufacturer: _TZE200_pay2byax
- Model identifier: TS0601

Clone
- _TZE200_kltffuzl
- _TZE200_fwoorn8y

### CODE REVIEW

File: devices/tuya/_TZE200_pay2byax_door_sensor.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": ["_TZE200_pay2byax", "_TZE200_kltffuzl", "_TZE200_fwoorn8y"],
+  "modelid": ["TS0601", "TS0601", "TS0601"],
+  "product": "ZG-102Z",
```
Comment (SwoopX) [inline devices/tuya/_TZE200_pay2byax_door_sensor.json:5]: I guess that should include the device type as well 🙂

File: devices/tuya/_TZE200_pay2byax_door_sensor.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
```
Comment (SwoopX) [inline devices/tuya/_TZE200_pay2byax_door_sensor.json:37]: Does it make sense here to add the usual Tuya firmware magic?

**Review by SwoopX — CHANGES_REQUESTED** [review]
@Smanar Some minors you might wanna check. Eventually, it makes sense to rename the json file to something like `ts0601_openclose_sensor.json`?

File: devices/tuya/_TZE200_pay2byax_door_sensor.json (hunk 1 above)
Comment (Smanar) [reply devices/tuya/_TZE200_pay2byax_door_sensor.json]: Renamed as _TZE200_pay2byax_openclose_sensor
All tuya stuff is using the manufacture name as file name https://github.com/dresden-elektronik/deconz-rest-plugin/tree/master/devices/tuya

>I guess that should include the device type as wel

You mean "product" ? I added some model

File: devices/tuya/_TZE200_pay2byax_door_sensor.json (hunk 2 above)
Comment (Smanar) [reply devices/tuya/_TZE200_pay2byax_door_sensor.json]: updated

**Review by SwoopX — APPROVED** [review]

**Pull request merged**